assets/html/documenter.css

```diff
@@ -105,6 +105,20 @@ pre code {
     background-color: initial;
 }
 
+kbd {
+    font: 11px "SFMono-Regular", Consolas, "Liberation Mono", Menlo, Courier, monospace;
+    display: inline-block;
+    padding: 3px 5px;
+    line-height: 10px;
```

Comment:
It would be great to have these in `em`s I think, so that it would scale properly with the rest. The border/shadow ones should stay `px` though.